### ISSUE TITLE
Tap on Selectboxes not working proper

### ISSUE THREAD

**Issue by christianarth** [tapquo/Lungo.js#47]
Hi,

I have a form with several input, select and button-elements.
When I try to tap on a selectbox, another will pop up or nothing happens.

It seems the clickable/tapable area is anywhere but not at the element.

Second issue:
When I select an option on a selectbox, the selectbox will not update its value.

The operating system is Android 2.2

Cheerz

**Comment by soyjavi** [comment]
What phone are you using? Could you paste the code here? Do you have a .scrollable class?

**Comment by christianarth** [comment]
I have tested it on an HTC Legend and HTC HD2.

Here is the code:

```
    <article id="add_data_main">
        <div id="add_data_main_scroller">
            <div class="form">
            <label>Amount</label>
            <input autocomplete="off" type="number" name="amount" id="fld_amount" value="0,00"  />

            <label>Title</label>
            <input autocomplete="off" type="text" name="title" id="fld_title" value=""  />

            <button id="fld_date" onclick="javascript:showcalendar();" data-month="" data-year="" data-week="">Datum</button>

           <label class="select">Recurring</label>
           <select name="cat" id="fld_recurring">
              <option value="false">Nein</option>
              <option value="monthly">Monatlich</option>
              <option value="yearly">Jährlich</option>
              <option value="three_months">Alle 3 Monate</option>
              <option value="six_months">Alle 6 Monate</option>
           </select>

            <label class="select">Categorie</label>
            <select name="cat" id="fld_cat"></select>

            <textarea id="fld_description" placeholder="Description"></textarea>
            </div>
        </div>
    </article>
```

And I have this scroll-view:

```
var properties = {
    hScroll: false,
    vScroll: true,
    useTransition: true,
    momentum: true,
    lockDirection: false,
    fixedScrollbar: true,
    fadeScrollbar: true,
    hideScrollbar: false,           
    onBeforeScrollStart: function (e) {
        var target = e.target;
        while (target.nodeType != 1){
            target = target.parentNode;
        } 
        if (target.localName != "select" && target.localName != "input" && target.localName != "textarea"){
            e.preventDefault();
        }
    }
};
```

lng.View.Scroll.create('add_data_main_scroller', properties);

**Comment by christianarth** [comment]
Even if i remove the labels, the problem exists when I add margin or padding to the elements.
If I press any input field it will generate a second input box that is placed above or below??

**Comment by soyjavi** [comment]
FIXED in Release 1.1

https://github.com/TapQuo/Lungo.js/commit/0a4142542093c2facf60389cd1b9ef14f5fbc685
